pkg/apis/extensions/v1alpha1/types.go

```diff
@@ -68,3 +71,17 @@ type Object interface {
 	// GetExtensionStatus retrieves the object's status.
 	GetExtensionStatus() Status
 }
+
+// ExtensionKinds contains all supported extension kinds.
+var ExtensionKinds = sets.NewString(
+	BackupBucketResource,
+	BackupEntryResource,
+	ContainerRuntimeResource,
+	ControlPlaneResource,
+	dnsv1alpha1.DNSProviderKind,
```

Comment:
What about `DNSEntry` and `DNSOwner`?

Reply:
Are these really extension Kinds we use for registration? Because we don't consider them [here](https://github.com/gardener/gardener/blob/8359e1ae278c1f121d1d3db69a96c1fa6e0ce4a5/pkg/operation/shoot/shoot.go#L465-L481) either.

Reply:
Yes, thanks, you're right!